app/package.json

```diff
@@ -15,8 +15,8 @@
     "@babel/core": "^7.0.0-0",
     "@babel/plugin-proposal-optional-chaining": "^7.14.5",
     "@nuxtjs/axios": "^5.13.6",
-    "@nuxtjs/sentry": "^5.1.5",
-    "@sentry/tracing": "^7.62.0",
+    "@nuxtjs/sentry": "^7.5.0",
+    "@sentry/webpack-plugin": "^1.20.1",
```

Comment:
Why is this needed?

Reply:
The @nuxtjs/sentry version bump solved our issue.
@sentry/webpack-plugin package could be used for the publishRelease function if we want to implement it in the future.
https://sentry.nuxtjs.org/guide/releases